backend/asmgen.ml

```diff
@@ -391,6 +398,8 @@ let compile_fundecl ?dwarf ~ppf_dump fd_cmm =
           ++ Profile.record ~accumulate:true "liveness" liveness
           ++ Profile.record ~accumulate:true "deadcode" Deadcode.fundecl
           ++ Compiler_hooks.execute_and_pipe Compiler_hooks.Mach_live
+          ++ Profile.record ~accumulate:true "polling"
+               (Polling.instrument_fundecl ~future_funcnames:funcnames)
```

Comment:
This pass already called in L362, why is it called again here?

Reply:
Well spotted, this is a merge error.